google/cloud/forseti/common/gcp_api/cloud_resource_manager.py

```diff
@@ -442,15 +436,14 @@ def get_folder(self, folder_name):
         except (errors.HttpError, HttpLib2Error) as e:
             raise api_errors.ApiExecutionError(folder_name, e)
 
-    def get_folders(self, resource_name, parent=None, show_deleted=False):
+    def get_folders(self, parent=None, show_deleted=False):
```

Comment:
Do we want to have the same interface with 
get_folders(self, parent=None, show_deleted=False)
get_projects(self, parent_id=None, parent_type=None, **filterargs)
?

Reply:
This is due to the differences in the underlying API calls for CRM v1 projects.list versus CRM v2 folders.list. Once there is a projects.list in v2 of the API, if it follows the format of taking a parent name instead of parent_id and parent_type then I'll update the interface.

The gcp_api libraries are meant to be relatively thin wrappers over the underlying API calls.